aws/data_source_aws_efs_file_system.go

```diff
@@ -22,6 +22,14 @@ func dataSourceAwsEfsFileSystem() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"availability_zone_id": {
```

Comment:
Hi @shuheiktgw 👋  Can you please add changelog entries for these new schema attributes? Reference: https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/pullrequest-submission-and-lifecycle.md#changelog-process

e.g. a new file called `.changelog/18319.txt` with contents:

``````markdown
```release-note:enhancement
data-source/aws_efs_file_system: Add `availability_zone_id` and `availability_zone_name` attributes
```

```release-note:enhancement
resource/aws_efs_file_system: Add `availability_zone_id` attribute and `availability_zone_name` argument
```
``````

Reply:
Sorry that I haven't noticed the new change log format! I'll fix it now!